charts/camunda-platform/README.md

```diff
@@ -908,6 +909,68 @@ elasticsearch:
+| | `image` |  Configuration for the Connectors image specifics | |
+| | `image.registry` | Can be used to set container image registry. | `""` |
+| | `image.repository` |  Defines which image repository to use | `camunda/connectors-bundle` |
+| | `image.tag` |  Can be set to overwrite the global tag, which should be used in that chart | `0.16.1` |
+| | `image.pullSecrets` | Can be set to overwrite the global.image.pullSecrets | `{{ global.image.pullSecrets }}` |
+| | `podAnnotations` | Can be used to define extra Connectors pod annotations | `{ }` |
+| | `podLabels` |  Can be used to define extra Connectors pod labels | `{ }` |
+| | `env` |  Can be used to set extra environment variables in each Connectors container | `[]` |
+| | `command` | Can be used to [override the default command provided by the container image](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/) | `[]` |
+| | `serviceAccount` |  Configuration for the service account where the Connectors pods are assigned to | |
+| | `serviceAccount.enabled` |  If true, enables the Connectors service account | `true` |
+| | `serviceAccount.name` |  Can be used to set the name of the Connectors service account | `""` |
+| | `serviceAccount.annotations` |  Can be used to set the annotations of the Connectors service account | `{}` |
+| | `service` |  Configuration for the Connectors service. | |
+| | `service.type` | Defines the [type of the service](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) | `ClusterIP` |
+| | `service.port` | Defines the port of the service, where the Connectors web application will be available | `8080` |
+| | `service.annotations` |  Can be used to define annotations, which will be applied to the Optimize service | `{}` |
+| | `podSecurityContext` | Defines the security options the Connectors pod should be run with | `{ }` |
+| | `containerSecurityContext` | Defines the security options the Connectors container should be run with | `{ }` |
+| | `nodeSelector` |  Can be used to define on which nodes the Connectors pods should run | `{}` |
+| | `tolerations` |  Can be used to define [pod toleration's](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) | `[ ]` |
+| | `affinity` |  Can be used to define [pod affinity or anti-affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) | `{ }` |
+| | `resources` | Configuration to set [request and limit configuration for the container](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) | `requests:`<br>`  cpu: 600m`<br> `  memory: 1Gi`<br>`limits:`<br> ` cpu: 2000m`<br> ` memory: 2Gi` || | `ingress` |  Configuration to configure the ingress resource | |
```

Comment:
The `resources` don't match what we actually have in the values file.

Reply:
Good catch. Updated.

Also removed the ingress since we're not using it right now.